server/src/building/BuildDatabase.cpp

```diff
@@ -357,13 +357,15 @@ std::shared_ptr<const BuildDatabase::TargetInfo> BuildDatabase::getClientLinkUni
 bool BuildDatabase::ObjectFileInfo::conflictPriorityMore(
         const std::shared_ptr<BuildDatabase::ObjectFileInfo> &left,
         const std::shared_ptr<BuildDatabase::ObjectFileInfo> &right) {
-    if (StringUtils::contains(left->getOutputFile().string(), "64")) {
+    bool leftContains64 = StringUtils::contains(left->getOutputFile().string(), "64");
+    bool rightContains64 = StringUtils::contains(right->getOutputFile().string(), "64");
+    if (leftContains64 && !rightContains64) {
         return true;
     }
-    if (StringUtils::contains(right->getOutputFile().string(), "64")) {
+    if (!leftContains64 && rightContains64) {
         return false;
     }
-    return false;
+    return left->getOutputFile().string() < right->getOutputFile().string();
```

Comment:
how is it related to the subject of improvement?

Reply:
Move to separate PR